salt/crypt.py

```diff
@@ -1417,13 +1417,13 @@ def sign_in(self, timeout=60, safe=True, tries=1, channel=None):
         auth["aes"] = self.verify_master(payload, master_pub="token" in sign_in_payload)
         if not auth["aes"]:
             log.critical(
-                "The Salt Master server's public key did not authenticate!\n"
+                "The Salt Master server's public key did not authenticate! "
                 "The master may need to be updated if it is a version of Salt "
-                "lower than %s, or\n"
+                "lower than %s, or "
                 "If you are confident that you are connecting to a valid Salt "
                 "Master, then remove the master public key and restart the "
-                "Salt Minion.\nThe master public key can be found "
-                "at:\n%s",
+                "Salt Minion. The master public key can be found "
+                "at: %s",
```

Comment:
**question** is this actually a related change? Seems like this would go better in a different PR.

Reply:
removed from this PR